Base/usr/share/man/man1/syscall.md

```diff
@@ -0,0 +1,58 @@
+## Name
+
+syscall - test a system call
+
+## Synopsis
+
+`syscall [-o] [-l] [-h] entry [args; buf==BUFSIZ buffer]`
```

Comment:
`entry` should be `<entry>` (as it's a required argument)

*What* is entry? A syscall name? Then maybe name it `<syscall-name>`? :smile:

Reply:
This should be a code block (not inline code). Like this:

    ```**sh
    $ syscall [-o] [-l] [-h] <syscall-name> [args; buf==BUFSIZ buffer]
    ```

Note the `**` before the `sh` (that's our little Markdown syntax extension) and the `$` (pseudo prompt) at the start.

Reply:
Hah, apparently my own older man pages don't follow the `<>` convention. Nevertheless, let's do it this way and fix those other man pages separately.